### DISCUSSION TITLE
[Announcement] Homework 6 is out

### DISCUSSION THREAD

**Discussion by yeonhee-ryou** [prosyslab-classroom/cs348-information-security#450] [Announcements]
Hi all,

This is an announcement about homework 6.
The goal of this homework is to implement a fuzzer that automatically generates inputs for software testing.

Here is the GitHub classroom link for this homework:
https://classroom.github.com/a/vDzYWwaP

## Auto grading

Your implementation will be graded using GradeScope.
Please check the guidelines
- [Evaluation criteria](https://docs.google.com/document/d/1k3f8-zILdUmiDNdKaBaBeu-H-MJmlqoJHMI8hPjBq3s/edit?usp=sharing)
- [GradeScope guideline](https://github.com/prosyslab-classroom/cs348-information-security/blob/main/TOOL.md#gradescope)

You can check the sample scores on the leaderboard.
- TA: elvis
- Other students of the previous classes: alumni1, alumni2, alumni3

## Due date

Due: 05. 31 (Wed.) 23:59:59
Late Due: 06. 02 (Fri.) 23:59:59 (late submission penalty)

**Comment by yeonhee-ryou** [comment]
This is **NOT** the last homework.
We still have one more homework, "HW7: Static Analyzer".
Please check the [schedule](https://github.com/prosyslab-classroom/cs348-information-security#schedule).

**Comment by adzaky15** [comment]
I keep getting autograder error. At first, I thought it is because of my implementation, but I tried uploading the code without mutation and the result is still the same. Can you please check this for me?

**Comment by snurf198** [comment]
@yeonhee-ryou Is it normal that my code takes very long time for gradescope to grade it? I wonder about this because since the program needs timeout to stop executing.

**Comment by KAIST-JongchanPark** [comment]
@adzaky15 It seems working well. In addition, I regrade your previous submission version and It works well. If you have same problem again, I recommend you to resubmit your submission.

**Comment by KAIST-JongchanPark** [comment]
@snurf198 It would take some time. Please check evaluation criteria.

@adzaky15 @snurf198 Please post your questions with new issue, not with comment.
